### PR TITLE
Initial readme

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,134 @@
...
+
+This packages contains macros to help you build your feature store right in dbt.
+
+# Usage
+
+## Inside of dbt Models
+
+> **NOTE:** to see a full example of the package in use, go to [dbt_feature_store_example](https://github.com/fal-ai/dbt_feature_store_example)
+
+You can models with these macros to maintain a feature store updated with your dbt runs.
```
Comment (turbo1912) [inline README.md]: build* models?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,134 @@
...
+
+## From fal client (coming soon)
+
+Trigger feature store calculations from a the fal client to iterate and test possible
```
Comment (turbo1912) [inline README.md]: Trigger feature calculations from a the fal python client to quickly iterate and discover the best features for your ML model from your notebook.

File: README.md [hunk 3]
```diff
@@ -1,15 +1,134 @@
...
-Try running the following commands:
-- dbt run
-- dbt test
+<!-- Added by: runner, at: Fri Feb 11 15:33:26 UTC 2022 -->
 
+<!--te-->
+
+# About
+
+This packages contains macros to help you build your feature store right in dbt.
```
Comment (turbo1912) [inline README.md]: This package contains dbt macros to help you build a feature store right in within your dbt repository.

File: README.md [hunk 4]
```diff
@@ -1,15 +1,134 @@
...
+
+
+## Building blocks Macros
+
+### next_timestamp ([source](/macros/next_timestamp.sql))
+
+Constructor: `feature_store.next_timestamp(entity_column, timestamp_column)`
+
+- `entity_column`: column name of id of rows for joining a label tables and feature tables
+- `timestmap_column`: column name of timestamp/date of rows for joining a label tables and feature tables
```
Comment (turbo1912) [inline README.md:88]: `timestamp_column`

"column name of the timestamp/date, this column is used to join labels to the features"

File: README.md [hunk 5]
```diff
@@ -1,15 +1,134 @@
...
+- `feature`: [feature_table object](#feature_table-object)
+
+
+## Building blocks Macros
+
+### next_timestamp ([source](/macros/next_timestamp.sql))
+
+Constructor: `feature_store.next_timestamp(entity_column, timestamp_column)`
+
+- `entity_column`: column name of id of rows for joining a label tables and feature tables
```
Comment (turbo1912) [inline README.md:87]: "column name of the entity id that is used for predictions, this column is used to join labels to the features"

File: README.md [hunk 6]
```diff
@@ -1,15 +1,134 @@
...
+
+### label_feature_join ([source](/macros/label_feature_join.sql))
+
+Constructor: `feature_store.label_feature_join(label_entity_column, label_timestamp_column, feature_entity_column, feature_timestamp_column, feature_next_timestamp_column)`
+
+- `label_entity_column`: column name of id of rows for joining a label tables and feature tables
+- `label_timestamp_column`: column name of timestamp/date of rows for joining a label tables and feature tables
+- `feature_entity_column`: column name of id of rows for joining a label tables and feature tables
+- `feature_timestamp_column`: column name of timestamp/date of rows for joining a label tables and feature tables
+- `feature_next_timestamp_column`: column pre-calculated (normally in a CTE) with the call of the macro [feature_store.next_timestamp(feature_entity_column, feature_timestamp_column)](#next_timestamp)
```
Comment (turbo1912) [inline README.md]: let's try to have the tone I had commented above here as well..

File: README.md [hunk 7]
```diff
@@ -1,15 +1,134 @@
...
+
+## From fal client (coming soon)
+
+Trigger feature store calculations from a the fal client to iterate and test possible
+features of interest before comitting with any specific one.
+
+
+# Macros
+
+## Plug-n-play Macros
```
Comment (turbo1912) [inline README.md]: do we need this plug-n-play header?

Comment (chamini2) [reply README.md]: Wanted to transmit that one is "very easy to use" and the others are building blocks for this one. How would you do this?

Comment (turbo1912) [reply README.md]: hmm i think no header for these two and `building blocks` for the others, but you can add a sentence explaining what create_dataset does;

`create_dataset` macro creates a table that holds the label and the historical features. This table should be ready to be used as training data without any additional transformations

**Review by turbo1912 — APPROVED** [review]

**Pull request merged**